monero-outreach-docs/translations/es/PSA/files/mining/mine_support_network_es.md

```diff
@@ -0,0 +1,27 @@
+# Minar Monero para apoyar la red
+
+Los mineros de Monero utilizan hardware informático para ejecutar una variante del algoritmo Cryptonight y competir por una recompensa asociado al siguiente bloque en la blockchain. La minería de Monero es reconocida por su resistencia a los ASIC. Esto quiere decir que es más accesible y amigable minar con CPU (unidad central de proceso) y hardware GPU (unidad de procesamiento gráfico), pero no garantiza la configuración que realizan los individuos y las compañias a los poderosos mineros ASICs y los FPGAs para minar Monero. Los desarrolladores de Monero han establecido una actualización rutinaria sobre la red, que tiene lugar cada 6 meses que ayuda a combatir en contra de la minería de Monero a través de los ASICs y los FPGAs y evitar que estos puedan obtener el control de la red a través del hashrate (poder de procesamiento).
```

Comment:
I can suggest to use _CryptoNight_ as is, with capital N.

Other suggestions:
- _...Monero es reconocida por su resistencia a dispositivos *ASIC*._
- _y competir por una recompensa *asociada*..._
- _...minar con *hardware* CPU (unidad central de proceso) y GPU..._ (re-structure of sentence).
- _...pero sin garantía contra individuos y compañias que configuran poderosos mineros ASICs y FPGAs para minar Monero._
- _... que tiene lugar cada 6 meses y ayuda a combatir contra los ASICs y FPGAs minando Monero y obteniendo control del poder de procesamiento de la red._

Reply:
done